gfop/get_food_counts.py

```diff
@@ -31,6 +31,32 @@ def get_sample_types(simple_complex='all'):
     return (gfop_metadata[['filename', *col_sample_types]]
             .set_index('filename'))
 
+def get_sample_metadata(gnps_network, all_groups):
+    '''
+    Args:
```

Comment:
**todo:** Start the docstring with a single sentence description of what the function does.